src/toil/utils/__init__.py

```diff
@@ -1 +1,46 @@
 from __future__ import absolute_import
+
+import os
+from toil import version
+from toil.lib.bioio import getBasicOptionParser, parseBasicOptions, setLoggingFromOptions
+import logging
+
+logger = logging.getLogger( __name__ )
+
+
+def getCurrentAWSZone():
```

Comment:
Now I am confused. Why doesn't _buildContext() use this? I left a bunch of comments there.